libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -166,18 +170,18 @@ void CompilerUtils::storeInMemoryDynamic(Type const& _type, bool _padToWordBound
 		m_context << Instruction::DUP2 << Instruction::MSTORE;
 		m_context << u256(_padToWordBoundaries ? 32 : 24) << Instruction::ADD;
 	}
-	else
+	else if (_type.isValueType())
 	{
 		unsigned numBytes = prepareMemoryStore(_type, _padToWordBoundaries);
-		if (numBytes > 0)
-		{
-			solUnimplementedAssert(
-				_type.sizeOnStack() == 1,
-				"Memory store of types with stack size != 1 not implemented."
-			);
-			m_context << Instruction::DUP2 << Instruction::MSTORE;
-			m_context << u256(numBytes) << Instruction::ADD;
-		}
+		m_context << Instruction::DUP2 << Instruction::MSTORE;
+		m_context << u256(numBytes) << Instruction::ADD;
+	}
+	else // Should not happen
+	{
+		solAssert(
+			false,
+			"Memory store of type " + _type.toString(true) + " not allowed."
+		);		
```

Comment:
There are trailing spaces here. I think we even have a CI test in latest develop that would detect it, but it did not run - did you rebase the branch onto latest develop?